### DISCUSSION TITLE
How to create a table in excel?

### DISCUSSION THREAD

**Discussion by ziongh** [sveinungf/spreadcheetah#62] [Q&A] [answered]
Firstly, great library! Nice to see a library independent of openxml, closed no, etc.

I've tried to export a table to excel using it, and it worked, however the exported table was not formatted as a table in excel (with the headers, with filtering, ordering, colors, etc). It appeared as a dump of the data into the excel.

Have I missed some config?

**Comment by sveinungf** [comment, marked as answer]
Thanks for the feedback!
Creating tables is not supported yet, unfortunately. It is something I want to add support for eventually though.
